spec/mongo/database_spec.rb

```diff
@@ -180,6 +180,24 @@
       it 'returns a list of the collections info' do
         expect(result).to include('acol')
       end
+
+      context 'with more than one collection' do
```

Comment:
There should be an assertion somewhere in this context that there is in fact more than one collection present in the database being used.